$NetBSD: patch-CMakeLists.txt,v 1.2 2025/01/17 17:44:22 rhialto Exp $

Doesn't respect CMAKE_INSTALL_MANDIR.
Add a command to terminate rxapi after it was started.
    https://sourceforge.net/p/oorexx/bugs/1993/

--- CMakeLists.txt.orig	2023-04-19 14:25:14.000000000 +0000
+++ CMakeLists.txt
@@ -290,7 +290,7 @@ else ()
    set (INSTALL_INCLUDE_DIR include)
    set (INSTALL_SAMPLES_DIR share/${CMAKE_PROJECT_NAME}/samples)
    set (INSTALL_DOC_DIR share/${CMAKE_PROJECT_NAME}/doc)
-   set (INSTALL_MAN_DIR share/man)
+   set (INSTALL_MAN_DIR ${CMAKE_INSTALL_MANDIR})
 endif ()
 
 # Set compiler and linker flags common to all build environments
@@ -1035,6 +1035,7 @@ endif ()
 # Build the rexx.img file
 add_custom_command(OUTPUT ${ORX_IMAGE_OUTPUT_LOCATION}/rexx.img
            COMMAND $<TARGET_FILE:rexximage> ${ORX_IMAGE_OUTPUT_LOCATION}/rexx.img
+           COMMAND $<TARGET_FILE:rxapi> stop
            DEPENDS rexximage rxapi ${image_class_files} ${platform_rexx_img_depends}
            WORKING_DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
 
